pennylane_lightning/core/src/simulators/lightning_kokkos/gates/BasicGeneratorFunctors.hpp

```diff
@@ -117,11 +112,10 @@ void applyGenCRZ(Kokkos::View<Kokkos::complex<PrecisionT> *> arr_,
 }
 
 template <class ExecutionSpace, class PrecisionT>
-void applyGenIsingXX(
-    Kokkos::View<Kokkos::complex<PrecisionT> *> arr_, std::size_t num_qubits,
-    const std::vector<std::size_t> &wires,
-    [[maybe_unused]] bool inverse = false,
-    [[maybe_unused]] const std::vector<PrecisionT> &params = {}) {
+void applyGenIsingXX(Kokkos::View<Kokkos::complex<PrecisionT> *> arr_,
+                     std::size_t num_qubits,
```

Comment:
do we want to add underscore suffix for an argument? I thought we usually add it to a private member.

Reply:
This is the convention already used throughout the `basicgatefunctors` and `matrixgatefunctors` (e.g. [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/9fc9633b10fb653f63b770f51785b90e56d78fda/pennylane_lightning/core/src/simulators/lightning_kokkos/gates/BasicGateFunctors.hpp#L68)). I believe here the convention is the objects without underscore are private members within the functor class (e.g. [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/9fc9633b10fb653f63b770f51785b90e56d78fda/pennylane_lightning/core/src/simulators/lightning_kokkos/gates/BasicGateFunctors.hpp#L37)), and underscore is added to these objects that gets passed into the functors. If the convention is to add `_` to private members instead, I can swap everything around; do we think this is a good idea? Should I do it here with this PR or as a separate PR?